blackbox/handlers/storage/dropbox.py

```diff
@@ -26,9 +27,25 @@ def __init__(self, **kwargs):
 
         self.upload_base = self.config.get("upload_directory") or "/"
         self.client = DropboxClient(self.config["access_token"])
+        self.valid = self._validate_token()
+
+    def _validate_token(self):
+        """Check if dropbox token is valid."""
+        try:
+            return self.client.check_user("test").result == "test"
```

Comment:
Sweet! Very nice that you found and used the dropbox clients built-in way of doing this! Extremely elegant solution.